### ISSUE TITLE
[Add/Edit Observation] Show user-defined "help text" on form questions

### ISSUE THREAD

**Issue by gino-m** [google/ground-android#800]
See also https://github.com/google/ground-platform/issues/704

**Comment by scolsen** [comment]
@JSunde assigning this to you for now, please reassign if needed!

**Comment by JSunde** [comment]
We're currently showing the Task "label", is that different from the "help text" that is being requested here?

**Comment by JSunde** [comment]
@gino-m

**Comment by gino-m** [comment]
Yes, this was intended to be should using a subtitle styling below the actual question. It could be used to provide additional instructions or clarifications. I don't think this is required for the pilots, so please feel free to unassign. In the meantime I'll move to our Icebox.
